### ISSUE TITLE
CreatedOn and ExpiresOn fields for test sessions appear to be inconsistent

### ISSUE THREAD

**Issue by AlexThurston** [usnistgov/ACVP-Server#159]
**environment**
Demo

**testSessionId**
N/A

**vsId**
N/A

**Algorithm registration**
N/A

**Endpoint in which the error is experienced**
Creating and reading a test session

It looks like there is a difference between in the format of the created on/expires on fields when you create a test session versus when you read a test session.  When I create a test session, the fields look as follows:
```
"createdOn": "2021-11-16T20:30:17.6014958-05:00"
"expiresOn": "2021-12-16T20:30:17.6014971-05:00"
```

And when I read the same test session:
```
"createdOn": "2021-11-16T20:30:35.12"
"expiresOn": "2021-12-16T20:30:35.12"
```

It'd be nice if the time format came back as something standard like perhaps RFC3339, or even Unix epoch time.  It's a little weird that these times are every so slightly different from each other, but perhaps this is because reading updates the expiry in demo.  That doesn't explain why the createdOn is different.

**Comment by livebe01** [comment]
Thanks @AlexThurston, we'll take a look.

**Comment by livebe01** [comment]
Hi @AlexThurston, I just took a look and the difference in timestamps between creating and reading a test session is due to our grabbing the time 2X in 2 separate places. We'll fix this.

Also, per [here](https://pages.nist.gov/ACVP/draft-fussell-acvp-spec.html#name-test-sessions) and [here](https://pages.nist.gov/ACVP/draft-fussell-acvp-spec.html#dateType), we should be using the RFC3339 UTC Zulu time when reporting timestamps. So, for example, "2021-11-16T20:30:17.6014958-05:00" should be coming back as "2021-11-17T00:30:17.6014958Z." We'll put in a fix for this as well.

**Comment by AlexThurston** [comment]
I just wondered if this was going to be address?  I just ran into an issue related to this.

**Comment by livebe01** [comment]
Thanks @AlexThurston. The issue you ran into, did it involve the use of inconsistent date/time formats?

**Comment by AlexThurston** [comment]
Yeah.  It did.  Basically I am parsing the `expiresOn` as a field the adheres to RFC3339 and when it doesn't, I end having to deal with an error.  In the meantime, I've pug a work around so that in the cases where I'm reading the test session, I won't assume it adhere's the RFC, but that is mostly just a stop-gap on my end.

**Comment by livebe01** [comment]
K, I appreciate you giving us a nudge on that. We'll fix the timestamp formats to be RFC3339 as they should be. 

We should fix the bit where we're pulling the time from two different places, but there are a number of moving parts w/ that... so I'm inclined to leave that alone for the moment.

**Comment by jbrock24** [comment]
This is fixed and will go out with the next update. Thanks for bringing this to our attention! Will close this once it's released.

**Comment by livebe01** [comment]
The fix for this is on Demo in release [v1.1.0.31](https://github.com/usnistgov/ACVP-Server/releases/tag/v1.1.0.31).

**Comment by livebe01** [comment]
The fix for this is on Prod in release [v1.1.0.31](https://github.com/usnistgov/ACVP-Server/releases/tag/v1.1.0.31).
